Dockerfile

```diff
@@ -30,7 +30,7 @@ ENTRYPOINT ["entrypoint.sh"]
 FROM base as builder
 ENV RAILS_ENV=production
 ENV FOREMAN_APIPIE_LANGS=en
-ENV BUNDLER_SKIPPED_GROUPS="test development openid libvirt journald facter"
+ENV BUNDLER_SKIPPED_GROUPS="test development openid libvirt journald facter jsonp console"
```

Comment:
I've been wondering about `jsonp`. Do people actually use this nowadays or can we deprecate it?

Reply:
I'm not sure, it was initially added for oVirt project, we did try to find out but didn't get a clear answer... maybe @shiramax can help track this down?

Reply:
Essentially it's requiring https://rubygems.org/gems/rack-jsonp which doesn't have any deps. In RPM/debs we unconditionally require it. I'm slightly leaning to keep it in the Docker builds since it should be very small and it does keep it consistent. In the mean time we should figure out a longer term plan perhaps?

Note I don't feel strongly about the inclusion/exclusion here, but would really like to know if we can get rid of it altogether.

Reply:
ACK, kept jsonp as it was before.